connectivity/tests/to-cidr.go

```diff
@@ -11,14 +11,20 @@ import (
 	"github.com/cilium/cilium-cli/connectivity/check"
 )
 
-// PodToCIDR sends an ICMP packet from each client Pod
-// to ExternalIP and ExternalOtherIP.
-func PodToCIDR() check.Scenario {
-	return &podToCIDR{}
+// PodToCIDR sends a HTTPS request from each client Pod
+// to ExternalIP default 1.1.1.1 and ExternalOtherIP default 1.0.0.1.
```

Comment:
Let's omit the defaults:

```suggestion
// to ExternalIP and ExternalOtherIP.
```